### PR TITLE
Refs # Fixed the web address

### LINKED ISSUE #18838: function help returns incorrect webpage
functionHelp() opens the associated help page for the function when selected by the user:

Just that the web address is lacking the final ".html" string, so the page does not display :stuck_out_tongue_winking_eye:

### PR DESCRIPTION
**Pull request by jmborr**
**To test:**
Open MantidPlot, add a function in the Fit Function dialog, right click to access the "Help" option, and then check your browser opens to the Help page...simple!

Fixes #18838.

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by AndreiSavici** [comment]
You should use `MantidQt::API::HelpWindow::showFitFunction(this->nativeParentWidget(),handler->ifun()->name());` instead. Also modify `MantidHelpWindow.cpp`, the `showFitFunction` function, so that the url points to `fitfunctions`, not `functions`

File: MantidQt/MantidWidgets/src/FitPropertyBrowser.cpp [hunk 1]
```diff
@@ -3,6 +3,7 @@
 #include "MantidQtMantidWidgets/SequentialFitDialog.h"
 #include "MantidQtMantidWidgets/MultifitSetupDialog.h"
 #include "MantidQtAPI/MantidDesktopServices.h"
```
Comment (AndreiSavici) [inline MantidQt/MantidWidgets/src/FitPropertyBrowser.cpp]: Do you need this line?

Comment (AndreiSavici) [inline MantidQt/MantidWidgets/src/FitPropertyBrowser.cpp]: Just ignore the comment. I see that it is used in `browserHelp`

**Review by AndreiSavici — APPROVED** [review]

**Comment by jmborr** [comment]
@AndreiSavici The builds finally passed. Can you take a final look, please?

**Comment by AndreiSavici** [comment]
:shipit:

**Comment by jmborr** [comment]
:barber: thanks :barber:

**Pull request merged**